yt_dlp/extractor/deuxm.py

```diff
@@ -58,8 +57,7 @@ class DeuxMNewsIE(InfoExtractor):
             'id': '634d9e108b70d40bc51a844b',
             'ext': 'mp4',
             'title': 'Interview: Casablanca hors des sentiers battus avec Abderrahim KASSOU (Replay) ',
-            'description': 'md5:3b8e78111de9fcc6ef7f7dd6cff2430c',
-            'thumbnail': 'https://2msoread-ww.amagi.tv/mediasfiles/videos/images/2022/10/17/1666031205e8b5613cf7fd24cc10a116844b23bfd4.png'
```

Comment:
why is this removed?

Reply:
should i add the regex to every thumbnail test?